aspnetcore/mvc/models/file-uploads.md

```diff
@@ -692,24 +692,32 @@ Other Kestrel limits may apply for apps hosted by Kestrel:
 * [Maximum client connections](xref:fundamentals/servers/kestrel#maximum-client-connections)
 * [Request and response data rates](xref:fundamentals/servers/kestrel#minimum-request-body-data-rate)
 
-### IIS content length limit
+### IIS content length and request body size limits
 
-The default request limit (`maxAllowedContentLength`) is 30,000,000 bytes, which is approximately 28.6MB. Customize the limit in the *web.config* file:
+The default request limit (`maxAllowedContentLength`) is 30,000,000 bytes, which is approximately 28.6 MB. Customize the limit in the `web.config` file. In the following example, the limit is set to 50 MB (52,428,800 bytes):
```

Comment:
code fenced web.config is a big improvement. We should try to do this everywhere.